.github/actions/validate-author/index.js

```diff
@@ -29,11 +51,13 @@ module.exports = async ({ context, github, core }) => {
         user: pullRequest.user,
       })
     ) {
-      core.setFailed(
-        `This workflow is not allowed to run on PRs from ${pullRequest.author_association}.`
-      );
+      const message = `This workflow requires repository owner, member, or collaborator permissions. Your current role (${pullRequest.author_association}) does not have sufficient permissions.`;
```

Comment:
Can we adjust the message? We should state that this workflow can only bw run on PRs file by a repository owner, member, or collaborator.